src/doc/gui_manual/Operators/OperatorTypes/PICS/Poincare_operator.rst

```diff
@@ -0,0 +1,250 @@
+   the winding pair 121:11 
+   the toroidal:poloidal periods (as a winding pair) 121:11 
+   the multiplication faction (11) i.e. diving by this number will give the base winding values, in this case 11:1. 
+   surface type: flux surface 
+   number of nodes in each winding group: with 4 nodes 
+   analysis state: complete. 
+
+
+Appearance
+^^^^^^^^^^
+
+The appearance tab specifies how the integral curve will be drawn. In addition to the Appearance attributes common to all PICS operators, the Poincare operator supports the following attributes.
+
+
+Data Value
+""""""""""
+
+Allows the user to set the data value associated with each data point and the overall limits
+
+None
+    Solid color from the single color 
+
+Safety Factor Q
+    Use the safety factor 
+
+Safety Factor P
+    Use the safety factor as defined when there are two possible choices for the magnetic axis 
+
+Safety Factor Q == P
+    Draw the surfaces on if the safety factor Q is equal to the safety factor P 
+
+Safety Factor Q != P
+    Draw the surfaces on if the safety factor Q is not equal to the safety factor P 
+
+Toroidal Windings Q
+    Use the toroidal winding value used in the calculation of Q 
+
+Toroidal Windings P
+    Use the toroidal winding value used in the calculation of P 
+
+Poloidal Windings
+    Use the poloidal winding value 
+
+Fieldline Order
+    Use input order of the seeds used to generate the fieldlines 
+
+Point Order
+    Use the puncture point index 
+Plane
+    Use the plane value (integer from 0 to N where N is the number of planes) 
+
+Winding Group Order
+    Use the winding group order (integer from 0 to T where T is the toroidal winding) 
+
+Winding Point Order
+    Use the index of the puncture points within each winding group 
+
+Winding Point Order Modulo Order
+    Use the order of the punctures within each winding group modulo the toroidal windings (useful for islands in islands) 
+
+
+Display
+"""""""
+
+Allows the users to display the results in a single plane or multiple planes. Further, one can reconstruct the 3D surface that the fieldline lies on.
+
+Mesh Type
+    Display the results as Curves or Surfaces.
+
+Overlapping Curve Sections
+""""""""""""""""""""""""""
+
+When displaying the data in a connected manner the raw data will often overlap itself. As such, for visually pleasing results it may be preferable to remove the overlaps.
+
+Raw
+    Display all of the punctures points in a connected fashion. 
+
+Remove
+    Display all of the punctures points in a connected fashion removing the overlapping sections. 
+
+Merge
+    Display all of the punctures points in a connected fashion merging the overlapping sections. Experimental. 
+
+Smooth
+    Display all of the punctures points in a connected fashion removing the overlapping sections while smoothing between points. Experimental. 
+
+
+Streamlines vs Pathlines
+""""""""""""""""""""""""
+
+See Streamlines vs Pathlines attributes that are common to all PICS operators.
```

Comment:
Make a link.

Reply:
streamlines, pathlines, fieldlines. Should we pick one and stick with it?

Reply:
Not sure what exactly fieldlines are, but streamlines and pathlines are used differently. The difference is explained [here](https://visit-sphinx-github-user-manual.readthedocs.io/en/docs-rusu1-visit-4445-document-integral-curves/gui_manual/Operators/OperatorTypes/PICS/index.html#streamlines-vs-pathlines)